examples/ola-streaming-client.cpp

```diff
@@ -41,6 +41,12 @@ DEFINE_s_string(dmx, d, "", "Comma separated DMX values to send, e.g. "
 DEFINE_s_uint32(universe, u, 1, "The universe to send data for");
 DEFINE_uint8(priority, ola::dmx::SOURCE_PRIORITY_DEFAULT,
              "The source priority to send data at");
+DEFINE_s_default_bool(universe_from_source, s, false,
+                      "Read universe number from source. Only allowed when "
+                      "processing STDIN. The universe number must be formatted "
+                      "in base 10, and be separated by at least one whitespace"
+                      " character from the start and end of the sequence of "
```

Comment:
Nit, we generally favour leaving the spaces at the end of a line as it makes it a bit more readable/them less likely to get dropped:
```suggestion
                      "in base 10, and be separated by at least one whitespace "
                      "character from the start and end of the sequence of "
```